cypress/integration/smoke.spec.ts

```diff
@@ -19,27 +19,31 @@ type ADXProvision = {
   datasources: ADXConfig[];
 };
 
+function addCommonProvisioningADXDatasource(ADXProvisions: ADXProvision[]) {
+  const datasource = ADXProvisions[0].datasources[1]; // Second entry in our common provisioning file
```

Comment:
I'm curious, does the first entry even work? (maybe we can delete it?)

Reply:
If we don't use the other entries, I'll delete all of them...

Reply:
Anyway the first one doesn't work.